css/style.css

```diff
@@ -62,18 +62,168 @@ body {
 /* Center the container */
 .container {
   width: 960px;
-  margin: auto;
-  height: 500px;
+  height: 700px;
+  float: center;
+  margin: 0 auto 0 auto;
+  padding: 40px;
+  background: url(../images/header_bg.jpg) no-repeat center top;
+}
+
+.benefits {
+  width: 960px;
+  height: 612px;
+  margin: 0 auto 0 auto;
+  padding: 80px;
+  background: #fff;
+  border: 0px;
+}
+
+.testimonial {
+  margin-top: 0px;
+  width: 960px;
+  margin: 0 auto 0 auto;
+  background: url(../images/testimonial_bg.jpg) no-repeat center top;
+}
+
+.yellow {
+  width: 960px;
+  height: 120px;
+  margin: 0px auto 0px auto;
+  background-color: #f9e42e;
+
+}
+
+p.quote {
+  margin: 0px;
+  padding: 110px 250px 120px 250px;
+  text-align: center;
+  font-style: italic;
+  font-weight: lighter;
+  color: #eee;
+}
+
+.images {
+  margin: 0px;
+  padding: 50px 250px 50px 250px;
+  text-align: center;
+  font-style: italic;
+  font-weight: lighter;
+  color: #eee;
+}
+
+p.copyright {
+  margin: 140px auto auto auto;
+  text-align: center;
+  font-style: normal;
+  font-size: 12px;
+  color: #eee;
+}
+
+
+.twitter {
+  float: left;
+  margin-left: 390px;
+  margin-top: 30px;
+}
+
+.facebook{
+  float: right;
+    margin-right: 400px;
+     margin-top: 30px;
+}
+    
+
+
+h1 {
+  font-size: 50px;
+  font-family: Helvetica, Arial, sans-serif;
+  font-weight: bold;
+  color: #eee;
+}
+
+h2 {
+  font-size: 50px;
+  font-family: Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  text-align: center;
+  margin-top: 100px;
+  color: #eee;
+}
+
+h3 {
+  font-size: 40px;
+  font-family: Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  text-align: left;
+  color: #013049;
 }
 
-header {
 
+p {
+  margin-top: 0px;
+  font-size: 26px;
+  text-align: left;
+  font-weight: lighter;
+  color: #666;
 }
 
-footer {
+ul {
+  width: 500px;
+  padding-left: 60px;
+  padding-top: 20px;
+  list-style-type: disc;
+  margin-top: 10px;
+  font-size: 28px;
+  text-align: left;
+  font-weight: lighter;
+  color: #666;
 
 }
 
+
+.button {
+    background-color: #f9e42e;
+    border: none;
+    color: #033049;
+    width: 260px;
+    height: 60px;
+    text-align: center;
+    text-decoration: none;
+    font-weight: bold;
+    margin-top: 70px;
+    margin-left: 330px;
+    font-size: 30px;
+    cursor: pointer;
+    border-radius: 5px;
+}
+
+.button2 {
+    background-color: #043049;
+    border: none;
+    color: #eee;
+    width: 304px;
+    height: 60px;
+    text-align: center;
+    font-weight: bold;
+    margin-top: 30px;
+    margin-left: 330px;
```

Comment:
:hammer: Looks like you're trying to center this button as well. Since it's in a `<p>`, you can remove the margin from the button and try styling the parent `<p>` to achieve centering the button.

There's another way too... remember how block elements take up the entire line? Essentially, a default width of 100%. What if we made the button's display block? Then we would just have to make sure the margin was "automatically" set equally on the left and right... (hint hint)